spec/acceptance/windows_spec.rb

```diff
@@ -5,17 +5,13 @@
   package_to_use = 'notepadplusplus.install'
   target_host = ENV['TARGET_HOST']
   before(:all) do
-    inventory_hash = inventory_hash_from_inventory_file
-    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'winrm_nodes')
-    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'local')
-    write_to_inventory_file(inventory_hash, 'inventory.yaml')
     command_string = 'cmd.exe /c puppet module install puppetlabs-chocolatey'
     command_string << " --modulepath #{Dir.pwd}/spec/fixtures/modules" if target_host.nil? || target_host == 'localhost'
     run_shell(command_string)
     pp = <<-PUPPETCODE
     include chocolatey
 PUPPETCODE
-    apply_manifest(pp)
+    apply_manifest(pp, expect_failures: true)
```

Comment:
we can now remove the expect_failures(since the chocolatey module is released with the fixes)